src/components/LogIn/LogIn.js

```diff
@@ -17,21 +28,25 @@ const LogIn = ({ children }) => {
     }
   });
 
+  const button = !useSelector(isConnecting)
+    ? generateButton("p-button", "_#", "Connecting...")
+    : generateButton(
+        "p-button--positive",
+        visitURL,
+        "Log in to view your models"
+      );
+
   if (!userIsLoggedIn) {
     return (
-      <div className="login">
-        <div className="login__inner">
-          <img className="login__logo" src={logo} alt="JAAS logo"></img>
-          <a
-            className="p-button--positive"
-            href={visitURL}
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            Log in to view your models
-          </a>
+      <>
+        <div className="login">
```

Comment:
I would prefer if this used `p-card--highlighted` with some positionally utilities but for a follow up.

Reply:
I've applied this class and made a few other tweaks to make it look better.